packages/react-vapor/src/components/table-hoc/tests/TableWithEmptyState.spec.tsx

```diff
@@ -77,4 +81,29 @@ describe('TableWithEmptyState', () => {
 
         jest.useRealTimers();
     });
+
+    it('should not call the setEmptyState from the TableHOCActions the table render has the setEmptyState property already set', () => {
```

Comment:
:thinking:  this title feels a bit wrong :stuck_out_tongue:

Reply:
Whoops, a few words are missing :sweat_smile: